data/src/config.rs

```diff
@@ -306,16 +306,30 @@ fn default_tooltip() -> bool {
 pub enum Error {
     #[error("config could not be read: {0}")]
     Read(String),
+    #[error("command could not be run: {0}")]
+    Execute(String),
     #[error("{0}")]
     Io(String),
     #[error("{0}")]
     Parse(String),
     #[error("UTF8 parsing error: {0}")]
-    UI(#[from] string::FromUtf8Error),
+    StrUtf8Error(#[from] str::Utf8Error),
+    #[error("UTF8 parsing error: {0}")]
+    StringUtf8Error(#[from] string::FromUtf8Error),
     #[error(transparent)]
     LoadSounds(#[from] audio::LoadError),
 }
 
+impl Error {
+    pub fn is_expected(&self) -> bool {
+        match self {
+            // If a user doesn't have a config when we start up, then we end up with a read error
+            Error::Read(_) => true,
+            _ => false,
+        }
+    }
+}
```

Comment:
`Error::Read(_)` isn't the best name anymore i'd say. Lets rename it now that you are touching these files, and while doing so rename this `fn` as well to make it more clear just by reading it.

Perhaps the error could be something like
`Error::Config(_)` or `Error::LoadConfigFile(_)` to keep it similar to `LoadSounds`.
*